bitnami/vault/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
This file has to be generated again to build the chart properly

Reply:
Fixed